markdownlint/docs/TOP009.md

```diff
@@ -0,0 +1,25 @@
+# `TOP009` - Lesson overview items sentence structure
+
+Tags: `content`
+
+Aliases: `lesson-overview-items-sentence-structure`
+
+This rule is triggered when a lesson overview item doesn't end with a period `.` or doesn't start with a capital letter. For example:
+
+```markdown
+- Lesson overview item 1
+- Lesson overview item 2?
+- lesson overview item 3.
+```
+
+To fix this issue, make sure that each bullet point ends with a period `.` and starts with a capital letter.
```

Comment:
Nit, even if the rational section does expand on it later.
```suggestion
To fix this issue, make sure that each bullet point ends with a period `.` and starts with a capital letter, rewording if you need to fix grammar since these should not be questions.
```